src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int side1;
+    private int side2;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int side1, int side2, int height) {
+        super(color);
+        this.side1 = side1;
+        this.side2 = side2;
+        this.height = height;
+    }
+
+    public double getArea() {
+        return 0.5 * (side1 * side2) * height;
```

Comment:
The formula to calculate the area of an isosceles trapezoid should be `0.5 * (side1 + side2) * height`, not `0.5 * (side1 * side2) * height`.